docker/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
I copied this from the ds3_c_docker repo, which needed it at the root leve.  removed.  (I think..?)